tests/unit/test_auth.py

```diff
@@ -0,0 +1,121 @@
+import os
+import tempfile
+from datetime import datetime, timedelta
+from unittest import mock
+from zoneinfo import ZoneInfo
+
+import jwt
+import pytest
+
+from IIIFingest.auth import Credentials
+
+
+def test_create_credentials_with_defaults():
+    issuer = "atdarth"
+    resources = ["ingest"]
+    kid = "atdarthdefault"
+    private_key = "secret"
+
+    creds = Credentials(
+        issuer=issuer, resources=resources, kid=kid, private_key_string=private_key
+    )
+
+    assert creds.issuer == issuer
+    assert creds.kid == kid
+    assert creds.private_key == private_key
+    assert creds.resources == resources
+    assert creds.algorithm == "RS256"
+    assert creds.expiration == 3600
+    assert creds.timezone == "America/New_York"
+
+
+def test_create_credentials_raises_error_without_private_key():
+    with pytest.raises(ValueError):
+        Credentials(issuer="atdarth", kid="atdarthdefault")
+
+
+@mock.patch.dict(os.environ, {"LTS_IIIF_ISSUER": "atdarth"}, clear=True)
```

Comment:
Neat, I don't recall seeing the use of `patch.dict` to modify `os.environ` before (or `clear=True`). Note that another way to approach this is pytest `monkeypatch`: https://docs.pytest.org/en/7.1.x/how-to/monkeypatch.html?highlight=environment.